packages/react-core/src/demos/Wizard/WizardDemo.md

```diff
@@ -117,9 +119,9 @@ import {
   DrawerPanelContent,
   DrawerHead,
   Modal,
-  ModalVariant,
-  Wizard
+  ModalVariant
 } from '@patternfly/react-core';
+import { Wizard as Wz } from '@patternfly/react-core/deprecated';
```

Comment:
nit.  I might have chosen a different alias name. Not a blocker though, just personal preference.